### PR TITLE
feat: document zeebe pause and resume exporting management api

### PR DESCRIPTION
**Pull request by deepthidevaki**
## What is the purpose of the change

- Document pause/resume exporting api of Zeebe. This api is required for the C8 backup process.

## Are there related marketing activities

_Are there any marketing activities related to the related feature or component? If so, please briefly describe them._

## When should this change go live?

_Does this change need to be released before a certain date or milestone? If so, when?_

## PR Checklist

- [ ] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [ ] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/self-managed/zeebe-deployment/operations/management-api.md
Comment (oleschoenburg) [inline docs/self-managed/zeebe-deployment/operations/management-api.md:7]: :wrench: I think it might be worth mentioning that these are spring actuators as that can point users in the right direction to configure security etc.

**Review by oleschoenburg — DISMISSED** [review]
:rocket:

File: sidebars.js
Comment (pepopowitz) [inline sidebars.js:704]: Due to the fact that we now have separate versions for Optimize than the rest of the docs, parallel sidebar changes will need to be made at https://github.com/camunda/camunda-platform-docs/blob/main/optimize_sidebars.js#L1312.

**Review by pepopowitz — CHANGES_REQUESTED** [review]
Additional changes are required to keep the sidebars from drifting in two separate areas of the site. 

See #1116 for more details on why these changes are needed, or let me know if I can further clarify!

File: sidebars.js
Comment (deepthidevaki) [reply sidebars.js:704]: Thanks. I have updated it.

**Comment by deepthidevaki** [comment]
@oleschoenburg Since technical writing review is already applied, could you please approve this PR? Then I can merge it to unblock the other PR.

**Review by oleschoenburg — APPROVED** [review]
> Since technical writing review is already applied

Oh is it? I thought we had to wait for `@christinaausley` for that.
I'm approving anyway :+1:

**Comment by deepthidevaki** [comment]
> > Since technical writing review is already applied
> 
> Oh is it? I thought we had to wait for `@christinaausley` for that. I'm approving anyway +1

I assumed @pepopowitz review covered it. Then I will wait before merging.

**Comment by akeller** [comment]
Sorry folks, I'm working through reviewing these PRs right now.

**Review by akeller — APPROVED** [review]
I added come review fixes in my latest commit. I'd also like to revisit this page for more linking across our docs, but I view that as non-blocking and something my team can take care of in the future. Thanks!

**Pull request merged**
